src/ra_server.erl

```diff
@@ -1104,8 +1111,16 @@ handle_follower({ra_log_event, Evt}, State = #{log := Log0}) ->
     % simply forward all other events to ra_log
     {Log, Effects} = ra_log:handle_event(Evt, Log0),
     {follower, State#{log => Log}, Effects};
+handle_follower(#pre_vote_rpc{},
+                #{cfg := #cfg{log_id = LogId}, voter := {no, _} = Voter} = State) ->
+    ?DEBUG("~w: follower ignored pre_vote_rpc, non-voter: ~p", [LogId, Voter]),
```

Comment:
use `~s` for the LogId and `~p0` for the Voter to ensure we avoid multi line logs.